tails_files/install.sh

```diff
@@ -39,19 +88,56 @@ chown root:root $SCRIPT_PY
 chmod 700 $SCRIPT_PY
 chown root:root $ADDITIONS
 chmod 400 $ADDITIONS
+
 chown amnesia:amnesia $INSTALL_DIR/securedrop_icon.png
-chmod 600 $INSTALL_DIR/securedrop_icon.png
-chown amnesia:amnesia $INSTALL_DIR/.xsessionrc
-chmod 700 $INSTALL_DIR/.xsessionrc
+chmod 600 $INSTALL_DIR/securedrop_icon.png 
+chown amnesia:amnesia $INSTALL_DIR/document.desktop $INSTALL_DIR/source.desktop $INSTALL_DIR/.xsessionrc
+chmod 700 $INSTALL_DIR/document.desktop $INSTALL_DIR/source.desktop $INSTALL_DIR/.xsessionrc
+
+# get addresses (journalist)
```

Comment:
Do we need zenity to do this? Once the journalist has filled out `torrc_additions`, can't we just extract this information (the interface .onion's) from there?